engine/src/flutter/shell/platform/windows/flutter_windows_engine.h

```diff
@@ -341,6 +348,14 @@ class FlutterWindowsEngine {
   // Allows swapping out embedder_api_ calls in tests.
   friend class EngineModifier;
 
+  // Maps a Flutter cursor name to an HCURSOR.
+  //
+  // Returns the arrow cursor for unknown constants.
+  //
+  // This map must be kept in sync with Flutter framework's
+  // services/mouse_cursor.dart.
```

Comment:
Could we update these lines too? (The UWP line is very stale and can be removed entirely, we forgot to do that when we dropped the UWP embedder)

https://github.com/flutter/flutter/blob/c38f0b7aaee3651f0eb98c7987da420f4225f504/packages/flutter/lib/src/services/mouse_cursor.dart#L411-L412